src/Tizen.NUI.Components/Controls/RecyclerView/Item/RecyclerViewItem.cs

```diff
@@ -50,6 +50,35 @@ public partial class RecyclerViewItem : Control
                     {
                         instance.UpdateState();
                     }
+                    if (instance.ParentItemsView is CollectionView colView)
```

Comment:
How about replacing the abbreviation variable name with a more appropriate one ? :)
`colView` ->`collectionView`

Reply:
thank you. I will update this with another patch.